ext/opcache/jit/zend_jit_x86.dasc

```diff
@@ -1612,8 +1612,7 @@ static int zend_jit_interrupt_handler_stub(dasm_State **Dst)
 	|	//if (EG(timed_out)) {
 	|	MEM_OP2_1_ZTS cmp, byte, executor_globals, timed_out, 0, r0
 	|	je >1
-	|	//zend_timeout(0);
-	|	xor FCARG1d, FCARG1d
```

Comment:
Has this line been removed deliberately?

Reply:
Yes. It's `zend_timeout()` and not `zend_timeout(0)`.

Do you know the `XOR %reg, %reg` trick for zeroing a register in assembly language? In many ISAs, that assembles to the smallest possible code for zeroing a register.

Reply:
Ah, yes, thanks!